test/e2e/v2/e2e_suite.go

```diff
@@ -159,7 +159,8 @@ var _ = Describe("kubebuilder", func() {
 
 			// NOTE: If you want to run the test against a GKE cluster, you will need to grant yourself permission.
 			// Otherwise, you may see "... is forbidden: attempt to grant extra privileges"
-			// $ kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@mycompany.com
+			// $ kubectl create clusterrolebinding myname-cluster-admin-binding \
+			// --clusterrole=cluster-admin --user=myname@mycompany.com
```

Comment:
```suggestion
			//     --clusterrole=cluster-admin --user=myname@mycompany.com
```

Reply:
1 of 100 comments requests to add \ and remove the spaces.
IHMO I prefer this other suggestion if you do not mind.

Reply:
Again, sorry.

Reply:
Now worry. Thank you a lot for your contributions btw.